bin/cassandra

```diff
@@ -166,7 +166,7 @@ case "`uname -s`" in
                 dirs="/lib64 /lib /usr/lib64 /usr/lib `cat /etc/ld.so.conf /etc/ld.so.conf.d/*.conf | grep '^/'`"
             fi
             dirs=`echo $dirs | tr " " ":"`
-            CASSANDRA_LIBJEMALLOC=$(find_library '.*/libjemalloc\.so\(\.1\)*' $dirs)
+            CASSANDRA_LIBJEMALLOC=$(find_library '.*/libjemalloc\.so\.*[1-9]*' $dirs)
```

Comment:
This will cover all libraries which are ending on a number, like:
````
/usr/lib/x86_64-linux-gnu/libjemalloc.so.1
/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
...
````